src/server/api/controllers/qr.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+
+const get = async (id) => {
+  try {
+    const instances = await knex('game_instances')
+      .select('game_instances.game_code as code')
+      .where({ id });
```

Comment:
Hi @masumesani! In the db the name of the column is `code`  not `game_code`. I changed the name and it worked on my local branch, but we need to make a decision because `code` is a reserved word for MySQL 8.0. 
We will review this together.

Reply:
I resolved this issue renaming to game_code in the commit d9a7a9705f33f4d29f35c4904372635a3746af5f